devenv/datasources.yaml

```
@@ -315,6 +315,13 @@ datasources:
       defaultRegion: eu-west-2
       customMetricsNamespaces: "CWAgent"
 
+  - name: JSON API - Journey Times
+    type: simpod-json-datasource
+    editable: true
+    url: http://0.0.0.0:8000/grafana/journey_times
```

left the same comment on Dorian's PR, I found it easier with this port being `8080` as its the SM default, and also so it doesn't collide with `audacity` running locally
```suggestion
    url: http://0.0.0.0:8080/grafana/journey_times
```